src/css/base/_variables_coa.scss

```diff
@@ -25,3 +25,11 @@ $coa-textwrap-width: 545px;
   height: $height;
   justify-content: space-between;
 }
+
```

Comment:
Not sure our colors are settled yet, but when they are, we should probably use these variables in _variables_uswds to overwrite primary, primary-alt and secondary colors.

Reply:
yeah agreed. I'm also thinking we might want to remove the USWDS dependency in a future refactor since there isn't a lot of championing of it within our team. I could be way off on this.